### PR TITLE
Collections Update 2023-02-15

### PR DESCRIPTION
**Pull request by adriannier**
- Swapped image URLs for Rolling Stone Greatest Albums from Rolling Stone server to Apple server
- Added Rolling Stone Concept Albums collection
- Made sure that there would be no regression regarding the Apple Music IDs in RS500.json

### CODE REVIEW

File: data/collections/RS500.json [hunk 1]
```diff
@@ -486,7 +486,7 @@
             "year": 1991,
             "appleMusicURL": null,
             "appleMusicID": null,
-            "imageURL": "https:\/\/www.rollingstone.com\/wp-content\/uploads\/2020\/09\/R1344-054-James-Brown-Star-Time.jpg"
+            "imageURL": null
```
Comment (nighthawk) [inline data/collections/RS500.json]: Might be worth keeping the URLs if they don't exist in Apple Music? Or could even host the somewhere?

File: _data/collections.yml [hunk 2]
```diff
@@ -15,4 +15,9 @@
   url: https://longplay.rocks/data/collections/NY100J.json
   count: 100
   date: 2008-05-19
+- name: "Rolling Stone: Concept Albums"
+  article_url: https://www.rollingstone.com/music/music-lists/best-concept-albums-1234604040/the-roots-20-1234604517/
```
Comment (nighthawk) [inline _data/collections.yml]:
```suggestion
  article_url: https://www.rollingstone.com/music/music-lists/best-concept-albums-1234604040/
```

The link was the the bottom of the page. Better to link to just the page.

**Review by nighthawk — COMMENTED** [review]
Nice one, just a suggestion to maybe keep the URLs of albums that aren't on Apple Music.

Something to consider for down the track would be to host images somewhere directly and so some processing on them, which the apps currently do direction: Namely they downscale them to get a thumbnail to use in the list view, and they also do colour analysis on the images. Only really needs to be done once server-side, rather than having each device to that.

File: data/collections/RS500.json (hunk 1 above)
Comment (adriannier) [reply data/collections/RS500.json]: What about copyright?

Comment (nighthawk) [reply data/collections/RS500.json]: Fair call. That's always a risk with album artwork I suppose. Safer to not host it.

(Though I have some in the screenshots anyway already.)

**Pull request merged**